src/firewall.py

```diff
@@ -66,7 +68,38 @@ def get_host_ip(self) -> str:
         )
         return str(ipaddress.IPv4Interface(address.strip()).ip)
 
-    def refresh_firewall(self, denylist: typing.List[FirewallEntry]):
+    def _exclude_network(
+        self,
+        networks: typing.List[NetworkT],
+        excludes: typing.List[NetworkT],
```

Comment:
Possibly rename to `exclude` since it already implies 1 or more

Reply:
Renamed, thank you!